src/js/App/GlobalFilter/TagsModal.tsx

```diff
@@ -129,27 +131,28 @@ const TagsModal = ({ isOpen, filterTagsBy, onApplyTags, toggleModal, selectedTag
       tableProps={{
         canSelectAll: false,
       }}
-      pagination={[tagsPagination, sidPagination]}
+      pagination={[tagsPagination, sidPagination] as TableWithFilterPagination[]}
       rows={[tagsRows, sidRows]}
-      loaded={[tagsLoaded, sidLoaded]}
+      loaded={[tagsLoaded as boolean, sidLoaded as boolean]}
       width="50%"
-      isOpen={isOpen}
-      toggleModal={(_e: unknown, isSubmit: boolean) => {
+      isOpen={isOpen as boolean}
+      toggleModal={(_e?: any, open?: boolean) => {
         setSidsSelected([]);
         setTagsSelected([]);
         setFilterBy('');
         setFilterSIDsBy('');
-        toggleModal(isSubmit);
+        toggleModal(open as boolean);
```

Comment:
Instead of the TS override can we do a `!!open` or in the argument make it optional with default value `_e?: any, open = false`?

Better than typecasting it.